src/main/Organization/OrganizationController.java

```diff
@@ -90,6 +94,130 @@ public OrganizationController(MongoDatabase db) {
         logger.info("Done with organizationSignupValidator");
       };
 
+  // Takes in a json object specifying usertypes and orgnames
+  //
+  //  {userTypes : [],
+  //  organizations : []}
+  public Handler findMembersOfOrgs =
+      ctx -> {
+        JSONObject ret = new JSONObject();
+
+        JSONObject req = new JSONObject(ctx.body());
+        JSONArray userTypes = req.getJSONArray("userTypes");
+        JSONArray orgs = req.getJSONArray("organizations");
+        if (userTypes.isEmpty()) {
+          logger.error("userTypes cannot be empty");
+          ctx.json(UserMessage.EMPTY_FIELD.toJSON().toString());
+          return;
+        }
+        MongoCollection<User> userCollection = db.getCollection("user", User.class);
+
+        logger.info("Counting usertypes");
+        if (orgs.isEmpty()) {
+          for (int i = 0; i < userTypes.length(); i++) {
+            String currType = userTypes.getString(i);
+            if (currType.equals("client")) {
```

Comment:
Anytime you make a long string of if else, esp if its checking a specific type, I would use a switch statement instead: 
switch(currType){
case "client"
case "worker"

etc.